bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -135,15 +136,17 @@ public ModbusSlave(String slave, KeyedObjectPool<ModbusSlaveEndpoint, ModbusSlav
      * writes data to Modbus device corresponding to OpenHAB command
      * works only with types "coil" and "holding"
      *
-     * @param command OpenHAB command received
+     * @param command openHAB command received
      * @param config
+     * @param readRegister
+     * @param writeRegister
      */
-    public void executeCommand(Command command, ModbusBindingConfig config) {
+    public void executeCommand(Command command, ModbusBindingConfig config, int readRegister, int writeRegister) {
         try {
             if (ModbusBindingProvider.TYPE_COIL.equals(getType())) {
-                setCoil(command, config);
+                setCoil(command, config, readRegister, writeRegister);
             } else if (ModbusBindingProvider.TYPE_HOLDING.equals(getType())) {
-                setRegister(command, config);
+                setRegister(command, config, readRegister, writeRegister);

```

Comment:
why the additional parameters `readRegister` and `writeRegister`? Shouldn't config include that?